### PR TITLE
MWPW-152270 - Performance updates and small changes to marketingtech.

### PR DESCRIPTION
**Pull request by bbythewa**
* Conditionally load assets from domain proxies to allow for better performance.
* Change default launch url for dev.

Resolves: [MWPW-152270](https://jira.corp.adobe.com/browse/MWPW-152270)

**Test URLs:**
- Before: https://stage--milo--adobecom.hlx.page/?martech=off
- After: https://marketingtech-updates--milo--adobecom.hlx.page/?martech=off

### CODE REVIEW

**Review by mokimo — APPROVED** [review]
Seems like one test is failing, apart of that this looks good.
```js
    it('have target be set to gnav and save in config', async () => {
      window.fetch = sinon.stub().returns(htmlResponse());
      document.head.innerHTML = await readFile({ path: './mocks/mep/head-target-gnav.html' });
      await utils.loadArea();
      const resultConfig = utils.getConfig();
      expect(resultConfig.mep.targetEnabled).to.equal('gnav');
    });
```

We need to make sure this doesn't get merged before the akamai changes are live though.

File: libs/martech/martech.js
Comment (robert-bogos) [inline libs/martech/martech.js:199]: This could be shorter
```suggestion
const isProxied = () => [
    'www.adobe.com', 'www.stage.adobe.com', 'milo.adobe.com',
    'milo.stage.adobe.com', 'business.adobe.com', 
    'business.stage.adobe.com', 'blog.adobe.com', 'blog.stage.adobe.com'
  ].includes(window.location.hostname)
```

**Review by robert-bogos — APPROVED** [review]
Looks good 👍 Just please make sure to check that failing unit test.
I left one small suggestion

File: libs/martech/martech.js
Comment (narcis-radu) [inline libs/martech/martech.js:188]: I think this would work better
```suggestion
  return /^(www|milo|business|blog)(\.stage)?\.adobe.com$/.test(hostname)
```

Comment (npeltier) [reply libs/martech/martech.js:188]: nit:  `return /^(www|milo|business|blog)(\.stage)?\.adobe\.com$/.test(hostname)
`

File: libs/martech/martech.js
Comment (bbythewa) [reply libs/martech/martech.js]: fixed

**Comment by bbythewa** [comment]
With the help of @chrischrischris , the tests are all passing now.  Thanks for the help!

All of the proxies are available on stage right now, so we can push this to stage, but we should NOT merge this to production until we have successfully tested all aspects of this as well as the proxies have been created. :) Thanks!

**Comment by mokimo** [comment]
@bbythewa on milo we don't have a great distinction of prod / stage. Stage is essentially "pre-prod" and just exists for testing reasons just before going live.
So, IMO the best course of action would be just testing if the proxies work on stage / prod.
Once akamai changes are on prod we merge this into stage, test and then it will automatically be promoted to prod by our automated process once testing finished.

**Comment by mokimo** [comment]
@bbythewa once you add the `Ready for Stage` label the process would merge this into stage on the next possible occasion automatically. Once in stage, you typically only have a day-ish with this residing on stage for regression testing before this goes onto prod.

The general advice is: Test on the feature branch, use [overrides](https://developer.chrome.com/docs/devtools/overrides) for any sort of testing on prod if you need and once testing has finished a QA should apply the `Ready for Stage` label which takes the code to (stage and) prod

**Pull request merged**
